### PR TITLE
[Issue-69] [Issue-107] Update UX home page and import account page

### PR DESCRIPTION
**Pull request by S2kael**
### Related issue

- #69 
- #107 

### Is your feature request related to a problem? Please describe.

- Update Import Account screen UX
- Update Home screen UX

### Describe the solution you'd like

- Change clickable of crypto info from all area to left area only
- Add smooth scrolling when crypto details are obscured
- Fix jerky with/without vertical scrollbar (set width for list crypto container)
- Reset password and error when change json file

### Describe alternatives you've considered
No

### Additional context
No

### CODE REVIEW

File: packages/extension-koni-ui/src/util/dom.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// [object Object]
```
Comment (lw-cdm) [inline packages/extension-koni-ui/src/util/dom.ts:1]: image
Incorrect header. Please run "yarn lint" for more details

File: packages/extension-koni-ui/src/util/dom.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+// SPDX-License-Identifier: Apache-2.0
+
+export const waitForElement = (selector: string, callback: (element: Element) => any) => {
+  let count = 0;
+  const poops = setInterval(function () {
```
Comment (lw-cdm) [inline packages/extension-koni-ui/src/util/dom.ts:6]: Change "poops" to another word.
Search Google for knowing what "poops" means.

File: packages/extension-koni-ui/src/Popup/Home/index.tsx [hunk 3]
```diff
@@ -464,7 +464,7 @@ export default React.memo(styled(Wrapper)(({ theme }: WrapperProps) => `
 
   .home-tab-contents {
     flex: 1;
-    overflow: auto;
+    overflow: overlay;
```
Comment (lw-cdm) [inline packages/extension-koni-ui/src/Popup/Home/index.tsx:467]: "overlay" does not work on Firefox. Please find another solution.
image

File: packages/extension-koni-ui/src/util/dom.ts (hunk 1 above)
Comment (lw-cdm) [inline packages/extension-koni-ui/src/util/dom.ts:1]: incorrect header when run yarn lint
image

File: packages/extension-koni-ui/src/util/dom.ts (hunk 2 above)
Comment (lw-cdm) [inline packages/extension-koni-ui/src/util/dom.ts:6]: change "poops" to another word. Do you know what the word mean ?

File: packages/extension-koni-ui/src/Popup/Home/ChainBalances/ChainBalanceDetail/ChainBalanceDetailItem.tsx [hunk 4]
```diff
@@ -44,7 +53,15 @@ function ChainBalanceDetailItem ({ accountInfo, balanceInfo, className, isLoadin
...
-  }, [accountInfo.networkKey, toggleBalanceDetail]);
+
+    if (!isShowDetail && toggleBalanceDetail) {
+      const callback = (element: Element) => {
+        element.scrollIntoView({ behavior: 'smooth', block: 'nearest' });
+      };
+
+      waitForElement(`.chain-balance-item__container.${networkKey}-detail`, callback);
+    }
+  }, [accountInfo.networkKey, toggleBalanceDetail, isShowDetail]);
```
Comment (lw-cdm) [inline packages/extension-koni-ui/src/Popup/Home/ChainBalances/ChainBalanceDetail/ChainBalanceDetailItem.tsx:64]: error when run yarn lint
image

**Review by lw-cdm — CHANGES_REQUESTED** [review]

**Review by sokol142 — COMMENTED** [review]
Still display message wrong password when change file. 
image

**Comment by S2kael** [comment]
> Still display message wrong password when change file. image

Fixed

**Review by lw-cdm — APPROVED** [review]
ok

**Pull request merged**